packages/models-library/src/models_library/services_db.py

```diff
@@ -34,6 +34,7 @@ class Config:
                 "thumbnail": "http://thumbnailit.org/image",
                 "created": "2021-01-18 12:46:57.7315",
                 "modified": "2021-01-19 12:45:00",
+                "deprecated": "2099-01-19 12:45:00",
```

Comment:
well I guess it is a WIP PR right? do you really want comments as of now?
if deprecated is a datetime, then define it as such. using ```str``` is easier but it is also a great pain later since anything can be in there, in any format, etc...

Reply:
true, it was WIP sorry.

I cannot comment on pros/cons of str and datetime. I have switched to datetime now as per your suggestion.